template/cordova/lib/build.js

```diff
@@ -171,11 +171,11 @@ function parseAndValidateArgs(options) {
 
     // Validate args
     if (options.debug && options.release) {
-        throw new CordovaError('Only one of "debug"/"release" options should be specified');
+        throw new CordovaError('Cannot specify "debug" and "release" options together. Please only use one');
```

Comment:
Nit: Error messages generally shouldn't say "please". I think you can omit this sentence anyway.